servers/rest/src/test/java/org/eclipse/iot/unide/server/web/RestEndpointPostgresTest.java

```diff
@@ -54,15 +55,21 @@
    public final WireMockRule wireMockRule = new WireMockRule( WIREMOCK_PORT );
 
```

Comment:
The test should renamed to RestEndpointSQLDatabaseTest.

Reply:
The sql statements are executed against wiremock and not against a database. 

Besides that, if HSQLDB does not support the json() function, the inseration of process message will fail. 

So we have to provide insert statements based on the actual database...